src/liballoc/tests/btree/map.rs

```diff
@@ -788,24 +787,35 @@ fn test_clone() {
         assert_eq!(map.len(), size / 2 - i - 1);
         assert_eq!(map, map.clone());
     }
+
+    // full 2-level and minimal 3-level tree
+    for i in 1..=144 {
+        assert_eq!(map.insert(i, i), None);
+        assert_eq!(map.len(), i);
+        if i >= 143 {
+            assert_eq!(map, map.clone());
+        }
+    }
 }
 
 #[test]
 fn test_clone_from() {
     let mut map1 = BTreeMap::new();
-    let size = 30;
+    let size = 12; // to obtain height 2 tree (having edges to leaf nodes)
 
-    for i in 0..size {
+    for i in 0..=size {
```

Comment:
So maybe add a comment here that we need `=` because we want the inner loop to run with `0..12`.